lib/charty/backends.rb

```diff
@@ -0,0 +1,32 @@
+module Charty
+  class BackendNotLoadedError < RuntimeError; end
+
+  module Backends
+    def self.register_backend(backend_class)
```

Comment:
We can omit `_backend` because this is `Backends` module.

Reply:
Done